.github/workflows/branches.yml

```diff
@@ -27,6 +27,8 @@ jobs:
           - etna
           - archimedes
           - deployer
+          - airflow
+          - swarm/swarm-keeper
```

Comment:
@corps , I figured I would sneak these in here. Is this correct for swarm-keeper? It seems to work when I run locally, but we don't have any other nested projects so wanted to check. Just `swarm` doesn't work...

Reply:
yeah, it's a good open question.  Let us just run airflow right now and not swarm-keeper.
In production and staging, the swarm images get built and released, but I do worry about having them run for every branch since they honestly aren't updated often.  It's just a question of CI performance vs completeness of branch testing.